docs/fields.asciidoc

```diff
@@ -1,6 +1,12 @@
 
-[[ecs-fields]]
-== {ecs} Fields
+[[ecs-field-reference]]
+== {ecs} Field Reference
+
+ECS defines multiple groups of related fields. They are called "field sets".
+The <<ecs-base,Base>> field set is the only one whose fields are defined
+at the top level of the events.
```

Comment:
"top level of the events."  would be clearer as  "top level of the ECS naming hierarchy."

Reply:
What about "whose fields are defined at the root of the event." ?

I would like it to be very clear at the conceptual level (it's just not nested).  I feel like this makes it clearer than talking about the ECS naming hierarchy. To someone new, it may not be obvious that the ECS hierarchy starts at the root of the document (it could be nested somewhere).